### ISSUE TITLE
Please make a new release

### ISSUE THREAD

**Issue by sthen** [uclouvain/openjpeg#782]
Numerous security bugs have been fixed since the last release, 2.1.0 (in 2014). Please consider making a new release so that OS packagers don't need to figure out what to backport (and quite possibly miss some things). Thank you!

**Comment by malaterre** [comment]
+1 

@detonin I do not believe the issue with RESTART mode needs to be addressed in this release. So I would just cut an iteration right now. Thanks.

**Comment by detonin** [comment]
ping @mayeut @malaterre 
I've modified appveyor and travis to ease the release process: now if a tag is pushed, it automatically creates a release and uploads windows, linux, and osx binaries to this release. This should allow us to release much more often.
However, before tagging v2.1.1, I would like the abi_check_report to be automatically added either to the source tree (like I did manually a few months ago here: https://github.com/uclouvain/openjpeg/blob/openjpeg-2.1/tools/abi-tracker/compat_report/2.1_to_current/compat_report.html) or to the release downloads. I did not yet have time to implement this but if one of you has time to do so, it's more than welcome.
By the way, currently we release only from openjpeg-2.x branches, not from master, is there any relevant reason to still do this ? It would be easier if we were tagging directly master.
Comments welcome.
